herddb-core/src/test/java/herddb/core/RestartTest.java

```diff
@@ -713,4 +713,93 @@ public void run() {
 
     }
 
+    @Test
+    public void recoverTableAndIndexWithoutHash() throws Exception {
+
+        Path dataPath = folder.newFolder("data").toPath();
+        Path logsPath = folder.newFolder("logs").toPath();
+        Path metadataPath = folder.newFolder("metadata").toPath();
+        Path tmoDir = folder.newFolder("tmpDir").toPath();
+
+        String nodeId = "localhost";
+        Table table;
+        Index index;
+
+        FileDataStorageManager.HASH_CHECKS_ENABLED = false;
```

Comment:
please use Writebox#setInternalState (update powermock to latest version in order to make it work with JDK12+)